server/socket/index.ts

```diff
@@ -64,10 +64,10 @@ export default function (
 
           // reset liveness timers
           if (ws.aliveTimer) clearTimeout(ws.aliveTimer);
-          ws.aliveTimer = setTimeout(() => { ws.isAlive = false; }, ALIVE_TIMEOUT);
+          ws.aliveTimer = global.setTimeout(() => { ws.isAlive = false; }, ALIVE_TIMEOUT);
 
           if (ws.expirationTimer) clearTimeout(ws.expirationTimer);
-          ws.expirationTimer = setTimeout(() => {
+          ws.expirationTimer = global.setTimeout(() => {
```

Comment:
why are these globals updated?

Reply:
I updated tsconfig adding esnext in lib to use a functionality which is not in es6. Then setTimeout made a type error. So I added it global then it is fixed.